### DISCUSSION TITLE
🎧 Kaizen 17 - Three wise men? - Dec. 13, 2024

### DISCUSSION THREAD

**Discussion by gerhard** [thechangelog/changelog.com#525] [Kaizen]
![badgen.net](https://flat.badgen.net/static/Last%20updated%20on%20December%2022,%202024/PUBLISHED/blue)

[🎧 This is now available as changelog.com/friends/73](https://changelog.com/friends/73) + [📝 episode transcript](https://github.com/thechangelog/transcripts/blob/master/friends/changelog--friends-73.md)

[![image](https://github.com/user-attachments/assets/ed0d75a0-4797-47fd-8317-0eb70943fa7b)](https://changelog.com/friends/73)

---

This contains all things related to the next Kaizen episode. [💬 There is a Zulip thread for this too](https://changelog.zulipchat.com/#narrow/channel/455097-kaizen/topic/Kaizen.20.2317).

Share your thoughts via comments below:
- Is there something in changelog.com that you, our dear listener, would like us to tackle?
- Is there something that **you** would like to help out with?
- Anything that we - @adamstac @jerodsanto - want to improve?

A rough timeline of what to expect (red line is today):

```mermaid
gantt
    title 🎧 Kaizen 17 timeline
    dateFormat  YYYY-MM-DD
    axisFormat %b %e, %Y

    section CONTRIBUTE
    💻 ALL CONTRIBUTIONS WELCOME :2024-09-20, 77d

    section RECORD

    🎙 Record :milestone, 2024-12-06
    ✂️ Edit :2024-12-06, 7d
    🚀 SHIP IT! :milestone, 2024-12-13

    section NEXT
    Kick-off Kaizen 18 :2024-12-13, 30d
```

---

### What changed? [Kaizen.16 ... Kaizen.17](https://github.com/thechangelog/changelog.com/compare/kaizen.16...kaizen.17)

- [CPU.fm](https://CPU.fm)
  - _The main thing is becoming the main thing again_
  - <https://github.com/thechangelog/cpu.fm>
  - _Bring back the HTML that was once lost_
- M4? Homelab?
- https://github.com/thechangelog/changelog.com/pull/533
- https://github.com/thechangelog/changelog.com/pull/534
- Zulip integrations
  - https://github.com/thechangelog/changelog.com/pull/535
  - https://github.com/thechangelog/changelog.com/pull/536
- Pipedream
  - https://github.com/thechangelog/pipedream/pull/3
  - Cache requests to feeds origin (R2)
  - 🧙🏼‍♂️🧙🏼‍♂️🧙🏼‍♂️

---

### Links for the show notes

- https://CPU.fm
  - https://github.com/thechangelog/cpu.fm
  - [📝 A new era for the Changelog Podcast Universe](https://changelog.com/posts/a-new-era-for-the-changelog-podcast-universe)
- https://pipely.tech 

---

### This is a follow-up to

- https://github.com/thechangelog/changelog.com/discussions/520
- https://github.com/thechangelog/changelog.com/discussions/511
- https://github.com/thechangelog/changelog.com/discussions/499

**Comment by adamstac** [comment]
Let's add [thechangelog/cpu.fm](https://github.com/thechangelog/cpu.fm) to the discussion.
